### DISCUSSION TITLE
Projekt eingestellt ?

### DISCUSSION THREAD

**Discussion by woodyBu** [Helium314/SCEE#697] [General]
Hi,

wurde das Projekt eingestellt ? Ich frage nur, weil es seit 8 Oktober kein Update mehr gibt, weder auf SC 59.3 noch auf 60.0.

wb

**Comment by rusty-snake** [comment]
:de: 

Ich denke nicht das es eingestellt wurde. @Helium314 hat in https://github.com/Helium314/HeliBoard/issues/598 darauf hingewiesen weitaus weniger Zeit zu haben. SCEE hat aber mehrere aktive Helfer (siehe commit history). Darum denke/hoffe ich das es weitergeht und wir gerade nur einen Tiefpunkt sehen in dem sich das Projekt neu orientieren und organiesieren kann.

Ansonsten müsste man mal darüber reden welche Features eine Zukunft in SC haben (ich hab schon eine Liste).

---

:gb: 

I don't think it has been discontinued. @Helium314 has pointed out in https://github.com/Helium314/HeliBoard/issues/598 to have much less time. But SCEE has several active helpers (see commit history). That's why I think/hope that it will continue and that we are just seeing a low point in which the project can reorient and reorganize itself.

Otherwise we should talk about which features have a future in SC (I already have a list).

Translated with DeepL.com (free version)

**Reply by Helium314** [reply]
> That's why I think/hope that it will continue and that we are just seeing a low point in which the project can reorient and reorganize itself.

Thanks, your guess is pretty much on point!
I'll be less active than previously though, as spending too much time on tasks that are more or less necessary, but not what I'm interested in, resulted in the need for a break.

**Comment by HolgerJeromin** [comment]
https://github.com/Helium314/SCEE/pull/680#issuecomment-2559270553

> Now that my break is over and I have a lot of stuff to catch up on...

Welcome back! :)
